src/modules/layouts/hooks/use-table.ts

```
@@ -8,33 +8,39 @@ import {
   columns,
   initialColumnOrder = columns.map((c) => c.id!),
   initialColumnVisibility = {},
   initialSorting = [],
-}: TableProps<T>): TableResponseType<T> => {
-  const { grouping } = useGrouping();
+}: TableProps<T>) => {
   const { filterGroup } = useFilterStore();
+  const { grouping, groupingType, groupingValue, groupingSort, createVisibilityManager } = useGrouping();
 
   const [globalFilter, setGlobalFilter] = useState("");
+  const [] = useState()
```

Biome (1.9.4)

[error] Unexpected empty array pattern.



(lint/correctness/noEmptyPattern)